### PR TITLE
Vcpkg

### PR DESCRIPTION
**Pull request by janisozaur**
This makes libraries available as created by vcpkg. Needs OpenRCT2/OpenRCT2#6792 .

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -4,17 +4,23 @@ cache:
...
 - cmd: install.bat
 build_script:
 - cmd: >-
-    build.bat x86
-
-    build.bat x64
-
-    package.bat
-
+    echo ok
```
Comment (IntelOrca) [inline appveyor.yml]: ?

**Comment by janisozaur** [comment]
Once the builds finish and artifacts are available, this should be ready to merge, but needs to have https://github.com/OpenRCT2/OpenRCT2/issues/6792 addressed.

**Comment by IntelOrca** [comment]
The zip contains a lot of things we don't need plus lots of artefacts rather than just the one we want. The zip should just be the directory containing `lib, bin, share, debug` etc.

**Comment by janisozaur** [comment]
isn't NuGet package superior? I thought you would be using that?

**Comment by IntelOrca** [comment]
I had a look and it isn't really superior. It is good if you want to provide a quick way to get a native library into a new project but otherwise no real benefit.

**Comment by IntelOrca** [comment]
There are two remaining issues:
* [ ] Debug builds fail due to the debug version of `discord-rpc` being built against /MT instead /MTd.
* [ ] libzip exposes its functions. In other words, `dllexport` was used when building libzip with static configuration.

**Pull request merged**